draft-ietf-tls-tls13.md

```diff
@@ -2236,9 +2232,12 @@ ffdhe2048, etc.
 
 Values within "obsolete_RESERVED" ranges were used in previous versions
 of TLS and MUST NOT be offered or negotiated by TLS 1.3 implementations.
+The obsolete curves have various known or theoretical weaknesses and are
+no longer considered safe for general use. The supported values are more
+than sufficient for interoperability with existing and future servers.
 
```

Comment:
I think "known or theoretical weaknesses" covers this rather well, though it's obviously a bit hand-wavey. They're not necessarily broken, just weak enough to be written out of the spec. All curves in the main obsolete set are either less than 255-bit (~128-bit security level), and thus weaker than the par we're setting for the spec, or a binary/Koblitz curve, which has at least theoretical weaknesses as was mentioned on the mailing list. The two in the second set are for arbitrary curves from RFC 4492, and can be configured to use whatever, including weak curves.

The "known or theoretical" wording is imprecise but not inaccurate. More information could be added here, but I don't think it'd be that useful.